__tests__/app.test.js

```diff
@@ -213,3 +213,53 @@ describe("/api/articles/:article_id/comments", () => {
       });
   });
 });
+describe("/api/articles/:article_id", () => {
+  test("PATCH: 201 updates article by article_id with positive vote increment", () => {
+    return request(app)
+      .patch("/api/articles/1")
+      .send({ inc_votes: 10 })
+      .expect(201)
```

Comment:
This code is for a successful "creation" a successful patch (updating something preexisting) can be a 200